Extensions/dnSpy.Analyzer/Properties/dnSpy.Analyzer.Resources.resx

```diff
@@ -159,6 +159,9 @@
   <data name="ImplementedByTreeNode" xml:space="preserve">
     <value>Implemented By</value>
   </data>
+  <data name="SubtypesByTreeNode" xml:space="preserve">
```

Comment:
I think we had a small miscommunication, the data name should be just “SubtypesNode” without the “By” word.